README.md

```diff
@@ -59,6 +59,122 @@ true
 false
 ```
 
+Remainder:
+```purs
+> 5.3 % 2.0
+1.2999999999999998
+```
+
+Trignometric functions:
+```purs
+> sin 0.0
```

Comment:
We have an opportunity to clarify that sin takes radians here; how about using pi/2 as the value?

Reply:
Fixed in 73c1e25368cd9e4165b4c143e6a618eb0ac52e92